KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteListPanelLayout.swift

```diff
@@ -9,27 +9,23 @@
 import FloatingPanel
 
 final class QuoteListPanelLayout: FloatingPanelLayout {
-
+    
     static let compactSize: CGFloat = 230
     private static let topOffset: CGFloat = 140
-
-    var initialPosition: FloatingPanelPosition {
-        return .half
+    
+    var position: FloatingPanelPosition = .bottom
+    var initialState: FloatingPanelState { .half }
+    var anchors: [FloatingPanelState : FloatingPanelLayoutAnchoring] {
+        return [.full: FloatingPanelLayoutAnchor(absoluteInset: 140, edge: .top, referenceGuide: .safeArea),
+                .half: FloatingPanelLayoutAnchor(absoluteInset: 230, edge: .bottom, referenceGuide: .superview),]
     }
 
-    var supportedPositions: Set<FloatingPanelPosition> {
+    var supportedPositions: Set<FloatingPanelState> {
         return [.full, .half]
     }
 
-    func insetFor(position: FloatingPanelPosition) -> CGFloat? {
-        switch position {
-        case .full: return QuoteListPanelLayout.topOffset
-        case .half: return QuoteListPanelLayout.compactSize
-        default: return nil
-        }
-    }
-
-    func backdropAlphaFor(position: FloatingPanelPosition) -> CGFloat {
+    //TODO: backdropAlphaFor is now backdropAlpha(for: FloatingPanelState) -> CGFloat which does not work with the current switch statement
```

Comment:
is that fixed now? seems like it should work

Reply:
Yeah, just left the comment there for reference in the future if it changes again according to the Migration guide from v1 to v2